pkgs/development/python-modules/koji/default.nix

```diff
@@ -2,12 +2,12 @@
 
 buildPythonPackage rec {
   pname = "koji";
-  version = "1.13.0";
+  version = "1.14.3";
   format = "other";
```

Comment:
still python2 only
```suggestion
  format = "other";
  disabled = !isPy27;
```

```
[nix-shell:~/.cache/nix-review/pr-72202]$ nix-shell --pure -p "with import ./nixpkgs {}; python3Packages.koji"

[nix-shell:~/.cache/nix-review/pr-72202]$ kojira --help
  File "/nix/store/5h0gb4z3614ndgskmyc2j888dd5dkw1d-python3.7-koji-1.14.3/bin/.kojira-wrapped", line 150
    except OSError, e:
                  ^
SyntaxError: invalid syntax
```

Reply:
Agh.

Reply:
Possibly the committer in e1182f070f11ee31e631d29bca16e2d7435cf3d7 was made overly optimistic from the `koji` executable appearing to work in py3k. Fixed.